src/baguetteBox.scss

```diff
@@ -32,8 +32,6 @@
 
         figure {
             display: inline;
-            margin: 0;
-            height: 100%; // Opera 12 image stretching fix
```

Comment:
I know how it looks like, but I'm sure that was a hack to fix some issue. It might have been Opera's engine bug, but I wouldn't place a special comment there if this wouldn't help.